helpinghand/locale/tr/LC_MESSAGES/django.po

```diff
@@ -43,6 +43,52 @@ msgstr "Oluşturulma Tarihi"
 msgid "Update Date"
 msgstr "Güncellenme Tarihi"
 
+#: helpinghand/apps/needs/apps.py:8 helpinghand/apps/needs/models.py:23
+msgid "Needs"
+msgstr "İhtiyaçlar"
+
+#: helpinghand/apps/needs/models.py:11
+msgid "Title"
+msgstr "Başlık"
+
+#: helpinghand/apps/needs/models.py:12
+msgid "Description"
+msgstr "Açıklama"
+
+#: helpinghand/apps/needs/models.py:13
+msgid "Address"
+msgstr "Adres"
+
+#: helpinghand/apps/needs/models.py:14
+#, fuzzy
+#| msgid "Update Date"
```

Comment:
```python
#, fuzzy
#| msgid "Update Date"
```

Bu kısmı silebilirsin. Sana yön göstermek için yazıyor. aşağıdaki çeviri buna benziyor gibi.